airflow/providers/google/cloud/utils/mlengine_operator_utils.py

```diff
@@ -203,7 +210,7 @@ def validate_err_and_count(summary):
     if dag is not None and dag.default_args is not None:
         default_args = dag.default_args
         project_id = project_id or default_args.get('project_id')
-        region = region or default_args.get('region')
+        region = region or default_args['region']
```

Comment:
When `default_args.get('region')` was being used, `mypy` said that the type was a `Union[str, Any, None]`.

This change also happens to have the fortunate side effect of [now matching what was already in the docstring](https://github.com/apache/airflow/blob/f618cdd19a5cdf09262436978817934d27668997/airflow/providers/google/cloud/utils/mlengine_operator_utils.py#L151).

Reply:
I would suggest `default_args.get("region", None)`

Reply:
I think region is actually required in the underlying operator and is expecting a string. Passing None as the second argument here is actually already the default behavior, too.

Reply:
Right, the `region` is necessary for `MLEngineStartBatchPredictionJobOperator`

Reply:
@turbaszek - if I'm reading the code correctly, that means that `None` won't work. Hence calling `default_args['region']` directly.